themes/survey/vanilla/css/theme.css

```diff
@@ -29,8 +29,14 @@
 */
 
 body {
-     padding-bottom: 10px;
+     padding-bottom: 0; /* no bottom padding, footer should be bottom aligned */
      padding-top: 60px;/* now is redefine in JS to fit any title length */
+     flex-direction: column; /* article, then footer */
+     display: flex; /* flex, to extend the content to full size */
```

Comment:
Using flex is a template choice, i didn't say it's a bad idea, but i don't think it must come from default lighter template.